operations/sat/sat_in_csm.md

```diff
@@ -0,0 +1,60 @@
+# System Admin Toolkit (SAT) in CSM
+
+The System Admin Toolkit (SAT) is a command-line interface that can assist administrators with common tasks, such as
+troubleshooting and querying information about the HPE Cray EX System, system boot and shutdown, and replacing hardware
+components. In CSM 1.3 and newer, the `sat` command is available on the Kubernetes NCNs without installing the SAT
+product stream.
+
+## SAT product stream components
+
+It is still possible to install SAT as a separate product stream. Installing the SAT product stream results in
+additional supporting components being installed.
+
+- Installing the SAT product stream creates an entry for SAT in the `cray-product-catalog` Kubernetes ConfigMap.
+  Without installing the SAT product stream, there will be no entry for this version of SAT in the output of
+  `sat showrev`.
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. It is still possible to uninstall
+  previous versions of the SAT product stream.
+
+- The `sat-config-management` git repository in Gitea (VCS) and thus the SAT layer of NCN CFS configuration is
+  only available with the full SAT product stream. See [SAT CFS configuration details](#sat-cfs-configuration-details)
+  below.
+
+- The `docs-sat` RPM package is only available with the full SAT product stream. SAT documentation can be found
+  at the links below (see: [SAT documentation](#sat-documentation)).
+
+### SAT CFS configuration details
```

Comment:
I'd suggest making this a level two heading like the others under "SAT in CSM."
```suggestion
## SAT CFS configuration details
```

Reply:
Would it be OK if I remove the heading entirely? Then this section will be nearly identical to the one we have in docs-sat.